pyproject.toml

```diff
@@ -72,22 +72,26 @@ relative_files = true
 [tool.coverage.report]
 exclude_lines = [
     "pragma: no cover",
-    "if TYPE_CHECKING:"
+    "if TYPE_CHECKING:",
+    "@overload",
```

Comment:
I think this should be `exclude_also` (also remove `pragma: no cover` if you change that).

Reply:
TIL, agree.